int32 total
int32 id
int32 time
float64 left_hip_yaw
float64 left_hip_roll
float64 left_hip_pitch
float64 left_knee
float64 left_ankle_pitch
float64 left_ankle_roll
float64 right_hip_yaw
float64 right_hip_roll
float64 right_hip_pitch
float64 right_knee
float64 right_ankle_pitch
float64 right_ankle_roll
float64 left_shoulder
float64 left_elbow
float64 right_shoulder
float64 right_elbow